stdlib/public/Differentiation/FloatingPointDifferentiation.swift.gyb

```diff
@@ -0,0 +1,275 @@
+//===--- FloatingPointDifferentiation.swift.gyb ---------------*- swift -*-===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2020 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import Swift
+import SwiftShims
```

Comment:
We might not need to import SwiftShims on master. What do you think?  Aside from that looks good.